src/nl802154.h

```diff
@@ -104,6 +106,12 @@ enum nl802154_attrs {
 
 	NL802154_ATTR_SUPPORTED_COMMANDS,
 
+	NL802154_ATTR_ADDRESS_MODE,
```

Comment:
I feel like addr_mode is already defined. The capabilities might be already defined. Short address and exte ded address are already defined.